roles/openshift_docker_gc/tasks/main.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create docker-gc tempdir
+  command: mktemp -d
+  register: templates_tmpdir
+
+# NOTE: oc_adm_policy_user does not support -z (yet)
+- name: Add dockergc as priviledged
+  shell: oc adm policy add-scc-to-user -z dockergc privileged
+#  oc_adm_policy_user:
```

Comment:
You can get the functionality of -z by specifying the entire path to the service account.
```
system:serviceaccount:<namespace>:dockergc
```